projects/go-lib/src/lib/components/go-tab/go-tab-group.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="go-tab__list">
+  <div
+    *ngFor="let tab of tabArray; let i = index"
+    class="go-tab__header"
+    [ngClass]="{ 'go-tab__header--active': tab.active, 'go-tab__header--dark': theme === 'dark' }"
+    (click)="activateTab(i)"
+  >
+    {{ tab.heading }}
+  </div>
+</div>
+
+<div class="go-tab__card--wrapper">
```

Comment:
It seems like this should probably not be a modifications, but maybe `go-tab__card-wrapper`?